plugin/trino-phoenix5/pom.xml

```diff
@@ -133,6 +133,14 @@
             </exclusions>
         </dependency>
 
+        <!-- Phoenix depends on log4j api and since it's banned, let's use hardened API-compatible fork -->
+        <dependency>
+            <groupId>ch.qos.reload4j</groupId>
+            <artifactId>reload4j</artifactId>
+            <version>1.2.25</version>
+            <scope>runtime</scope>
+        </dependency>
```

Comment:
seems very brittle

Reply:
What's the alternative? Ideas?

Reply:
Actually looks like Phoenix should already be pulling in reload4j and the slf4j shim for it. See https://github.com/apache/phoenix/commit/cd35c5239bd7e3f263d3f2a99ce1b996205bde4b but I don't know what version that commit is a part of.

Reply:
I'm afraid it's not the case for 5.1.3 :(